webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Thermal/utils.ts

```diff
@@ -167,28 +108,103 @@ export const POLLUTANT_NAMES: Array<keyof PollutantFields> = [
 // Functions
 ////////////////////////////////////////////////////////////////
 
-function makeRequestURL(
+const CLUSTERS_URL = (
```

Comment:
No uppercase and underscore for functions

Reply:
my bad thanks